test/Microsoft.Azure.WebJobs.Host.UnitTests/Timers/TaskSeriesTimerTests.cs

```diff
@@ -289,6 +289,19 @@ public void Start_IfStarted_Throws()
             }
         }
 
+        [Fact]
+        public void Start_IfStopped_DoesNotThrow()
+        {
+            // Arrange
+            ITaskSeriesCommand command = CreateStubCommand(TimeSpan.Zero);
+            ITaskSeriesTimer product = CreateProductUnderTest(command);
+            product.Start();
+            product.StopAsync(CancellationToken.None).GetAwaiter().GetResult();
```

Comment:
Instead of using a blocking pattern here, please make the test `async Task` instead and await the stop :)

Reply:
Done and cleaned all the tests file